src/CoreWCF.Http/src/CoreWCF/HttpTransportSecurity.cs

```diff
@@ -80,13 +80,13 @@ void ConfigureAuthentication(HttpTransportBindingElement http)
             //http.ExtendedProtectionPolicy = extendedProtectionPolicy;
         }
```

Comment:
I think this line needs to be uncommented as you have lit up the ExtendedProtectionPolicy code